consensus/consortium/v1/consortium.go

```diff
@@ -485,7 +484,8 @@ func (c *Consortium) Prepare(chain consensus.ChainHeaderReader, header *types.He
 	header.Extra = header.Extra[:extraVanity]
 
 	if number%c.config.Epoch == 0 {
-		validators, err := c.getValidatorsFromContract(chain, number)
+		var validators []common.Address
+		validators, err := c.getValidatorsFromGenesis()
```

Comment:
This is incorrect. getValidatorsFromGenesis is only the same as getValidatorsFromContract at genesis block. The if case here is meant to be triggered every checkpoint. The Prepare is not important in sync process so I think we don't need to change this.

Reply:
hmm, agree already tested again for verifying, seem prepare is only triggred when mining new block, so this logic will not happen when sync logic, so  I will keep the logic, only return data from genesis in snapshot and last checkpoint logic.